dev/com.ibm.ws.jsp/src/com/ibm/ws/jsp/runtime/PagesDirectiveImportInfo.java

```diff
@@ -14,10 +14,21 @@
 
 import java.util.List;
 
-public interface JspImportInfo {
+/**
+ * Interface for retrieving page directive information as well as page import information.
+ */
+public interface PagesDirectiveImportInfo {
```

Comment:
```suggestion
public interface PagesDirectiveInfo {
```

I think this should be a source of all directive information, so  I think it makes sense to remove `Import` from the name.

Reply:
Agh. Then again, these directives also apply to Tags. Maybe the name should just be DirectiveInfo?